src/vs/workbench/browser/parts/editor/editorGroupsControl.ts

```diff
@@ -86,6 +89,23 @@ export interface IEditorGroupsControl {
 	dispose(): void;
 }
 
+function toSize(sizeStr: string, maxSize: number, defaultSize: number): number {
```

Comment:
Any reason why this method is not a private method within the `EditorGroupsControl` close to where it is being used?